glib/src/subclass/signal.rs

```diff
@@ -12,11 +12,11 @@ use std::{fmt, num::NonZeroU32};
 
 /// Builder for signals.
 #[allow(clippy::type_complexity)]
-pub struct SignalBuilder<'a> {
+pub struct SignalBuilder<'a, T: Into<SignalType> + Clone> {
```

Comment:
That makes not much sense. You will usually use `Type` for most things and a `SignalType` for one parameter or two, but you can't mix those two here and will get confusing errors.

I'd make this un-generic and use `SignalType` everywhere

Reply:
Yes, I wanted to avoid having to do .into() everywhere where I had `T::static_type()` before. I have made it un-generic now